File: sdkwork-backend-sdk-go/types/plus_department_vo.go
package types

// Department VO
type PlusDepartmentVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Uuid string `json:"uuid"`
	ParentId int `json:"parentId"`
	Name string `json:"name"`
	Description string `json:"description"`
	Owner string `json:"owner"`
	OwnerId int `json:"ownerId"`
	Code string `json:"code"`
	SortOrder int `json:"sortOrder"`
	IsActive bool `json:"isActive"`
	OrganizationId int `json:"organizationId"`
	MemberIds []int `json:"memberIds"`
}

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_department_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusDepartmentVO:
    """Department VO"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    uuid: str = None
    parent_id: int = None
    name: str = None
    description: str = None
    owner: str = None
    owner_id: int = None
    code: str = None
    sort_order: int = None
    is_active: bool = None
    organization_id: int = None
    member_ids: List[int] = None
